src/index.html

```diff
@@ -1,110 +1,117 @@
+                    <span class="main-text-body">Your virtual DJ with a democratic playlist ♫</span><br>
+                    <span class="main-text-body" id="main-text">It looks like you don't have Javascript enabled.<br>
+                            However, it is needed to join or create a JamSession.</span>
+                </div>
+            </div>
 
-        <div class="main-create-container">
-            <button class="button button-green" id="login-button">Login to Spotify to create your own</button>
-        </div>
+            <div class="main-content-container d-flex jy-around al-center flex-column">
+                <div class="main-join-container w-100 d-flex jy-center al-center">
+                    <input class="jamlabel-input h-100 w-100" id="jamlabel-input"
+                           maxlength="5"
+                           placeholder="Enter a JamLabel..." required type="text">
+                    <div class="main-join-button-container">
+                        <button class="button button-orange button-big disabled" id="join-button">
+                            Join
+                        </button>
+                    </div>
+                    <span class="jamlabel-input-error" hidden id="join-error">Error message</span>
+                </div>
+                <div class="main-button-divider d-flex jy-center al-center">
+                    <hr>
+                    or
+                    <hr>
+                </div>
+                <div class="main-create-container">
+                    <button class="button button-green login-button" id="login-button">Login to Spotify
+                        to create your own
+                    </button>
+                </div>
+            </div>
+        </main>
 
-    </div>
-</main>
-</body>
+        <footer class="d-flex jy-between al-center w-100">
+            <div class="github-container h-100 d-flex jy-center al-center">
+                <svg height="20" width="20" aria-hidden="true" viewBox="0 0 16 16" version="1.1" data-view-component="true" class="octicon octicon-mark-github v-align-middle">
+                    <path fill="white" fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"></path>
+                </svg>
+                <a href="https://github.com/JamFactoryApp/jamfactory-web/" target="_blank">JamFactory</a>
+            </div>
+            <a href="/imprint.html" target="_blank" class="h-100 impressum">Impressum</a>
```

Comment:
I would remove the ``target="_blank"`` from the link to the imprint. The imprint should not open a new tab.
However with the current  ``transition: 0.5s;`` for everything this looks kinda strange.

See updated Readme for information about untracked files.